Library/ViewModels/SettingsAccountViewModel.swift

```diff
@@ -147,6 +151,11 @@ SettingsAccountViewModelOutputs, SettingsAccountViewModelType {
     self.viewDidLoadProperty.value = ()
   }
 
+  fileprivate let userHasPasswordAndEmailProperty = MutableProperty<(Bool, String?)>((true, nil))
```

Comment:
Should we test this?

Reply:
Also instead of using default value `(true, nil)`, would it be safer to use optional tuple that would not be initialized like this?

So instead of
```
fileprivate let userHasPasswordAndEmailProperty = MutableProperty<(Bool, String?)>((true, nil))
```
use
```
fileprivate let userHasPasswordAndEmailProperty = MutableProperty<(Bool, String)?>(nil)
```

Reply:
In this particular case, we are `returning` a MutableProperty value and not `assigning` a value, like we usually do for functions. So we would have to set a default value anyway, since the variable is non-Optional. Does it make sense? 
Based on your previous comment though, I would change to <(Bool, String)> instead, making the String non-optional.

What do you think?

Reply:
Thanks for the explanation Nino! It makes perfect sense now 😄 

![giphy](https://media.giphy.com/media/vEKpd4qBcgqHe/giphy.gif)